docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
 ---
-title: Getting Started with Desktop Access
-description: Passwordless access to Windows systems for local users
+title: Get started with Desktop Access
+description: Configure passwordless access to Windows computers for local users.
 videoBanner: 9DyKQbg4ORc
 ---
 
-This guide will help you configure Teleport to provide secure, passwordless access
-to Windows desktops. This configuration does not require an Active Directory domain.
+This guide demonstrates how to configure Teleport to provide secure, passwordless access
+to Microsoft Windows desktops that aren't joined to an Active Directory domain.
+Passwordless access is only supported for **commercial editions** of Teleport.
+For the open source Teleport Community Edition, you can protect Microsoft Windows servers 
+and desktops by integrating Teleport with Active Directory. 
 
-<Admonition type="note">
-Passwordless access for local users is an Enterprise-only feature.
+Integrating with Active Directory enables you to automatically discover Microsoft Windows servers 
```

Comment:
Most of the text in the introductory section of this page is about why _not_ to follow this page, which reads a bit oddly to me. Only the first sentence of the introductory section has to do with this guide.

I think it would make sense to make this an H3 within the Prerequisites section, since having a commercial edition is technically a prerequisite.

Reply:
To be honest, I've never felt great about this page. It started out describing the AD approach, since that was all we supported, and we named it "getting started."

Then we released support for non-AD, but for enterprise only, and we decided to make the "getting started" page describe that flow instead (while the AD content got moved to a new page). But it's not clear from the nav or the title that this is the non-AD guide.

I think we probably want to make the getting started page much smaller and just provide a brief intro of the stuff that is common to both approaches (joining the agent, etc), with links to the separate AD/non-AD pages.

Reply:
I was going at this with blinders on, but I think I need to look at this whole section to do something reasonable. Can we retire the Active Directory (Manual) page? It seems like it refers to a procedure for pre-10.2.6. It feels confusing to me to include it as a red herring for people to trip on.

Reply:
I don't think we can remove it, because:

1. Some people don't trust the automatic scripts and want to do the configuration step-by-step.
2. Sometimes the automatic scripts aren't flexible enough and the customer has to make the changes manually.

Reply:
@zmb3 I changed the name of this non-AD topic and moved some material into the introduction. The file name change had a ripple effect on other files. I also removed the caveat about commercial only support. 
If you get a chance, could you take another look?